x-pack/plugins/siem/public/components/tables/helpers.test.tsx

```diff
@@ -87,6 +103,21 @@ describe('Table Helpers', () => {
       expect(wrapper.html()).toBe(getEmptyValue());
     });
 
+    test('it returns empty string value when rowItem is empty', () => {
```

Comment:
I've got my 👀on `getRowItemDraggable` & `getRowItemDraggables`. If they don't end up delineating from each other as we iterate on the table/field renderers, we may want to consolidate them into a single component that supports both `string` | `string[]` that way we don't have to support additional tests for the same functionality.

Nothing to change here now -- just where my heads been at in regards to all the new field renderers popping up (e.g. similar renderers for HostOverview as well).